i18n/fr.yaml

```
@@ -264,52 +263,58 @@ homepage-used-by-hundred-list-four:
 homepage-used-by-image-alt:
-  other: "Une personne debout devant une image de la Terre. Autour d’elle se trouvent des portraits de gens connectés par un réseau de lignes."   
+  other: "Une personne debout devant une image de la Terre. Autour d’elle se trouvent des portraits de gens connectés par un réseau de lignes."
 contact-cds:
-  other: "Gardons le contact"          
+  other: "Gardons le contact"
 newsletter:
   other: "Infolettre"
 linkedin:
   other: "FR LinkedIn"
 footer-heading:
   other: "Service numérique canadien (SNC)"
 footer-contact-us:
-  other: "Contactez le SNC"  
+  other: "Contactez le SNC"
 footer-jobs:
   other: "Emplois au SNC"
 footer-jobs-link:
-  other: "/emplois/"   
+  other: "/emplois/"
 footer-security:
   other: "Sécurité"
 footer-security-link:
-  other: "/avis-de-securite/"                 
+  other: "/avis-de-securite/"
 footer-privacy:
   other: "Confidentialité"
 footer-privacy-link:
-  other: "/confidentialite/"  
+  other: "/confidentialite/"
 footer-terms:
   other: "Avis"
 footer-terms-link:
-  other: "/avis/"  
+  other: "/avis/"
 footer-visit-canada-ca:
   other: "Visitez Canada.ca"
 footer-visit-canada-ca-link:
-  other: "https://www.canada.ca/fr.html"  
+  other: "https://www.canada.ca/fr.html"
 esdc:
   other: "Emploi et Développement social Canada"
 aa-language:
-  other: "fra"  
+  other: "fra"
 meta-description:
   other: "Service numérique canadien, Des produits numériques pour le gouvernement du Canada, Envoyez des courriels et des messages texte automatisés, Créez des formulaires en ligne accessibles, Composants réutilisables conformes aux normes du Web, Des outils créés par le gouvernement, pour le gouvernement"
 meta-keywords:
-  other: "Service numérique canadien, produits de plateforme, Notification GC, Formulaires GC, GC Design System, Articles GC, Générateur de confidentialité, Alerte COVID, Explorez les avantages et les services, Les justificatifs numériques,"                        
+  other: "Service numérique canadien, produits de plateforme, Notification GC, Formulaires GC, GC Design System, Articles GC, Générateur de confidentialité, Alerte COVID, Explorez les avantages et les services, Les justificatifs numériques,"
 meta-title:
   other: "Page d'accueil"
 meta-identifier:
   other: "Service_numérique_canadien"
 meta-subject:
-  other: "GC numérique, Service numérique canadien, services numériques, numérique du gouvernement, plateform, source ouverte; accessibilité, nuage, API, egile, essais,"                                 
+  other: "GC numérique, Service numérique canadien, services numériques, numérique du gouvernement, plateform, source ouverte; accessibilité, nuage, API, egile, essais,"
 homepage:
   other: "Page d’accueil"
 ga4-linkedin:
-  other: "FR_LinkedIn"        
\ No newline at end of file
+  other: "FR_LinkedIn"
+tagged-with:
+  other: "Balisé avec&nbsp;"
+topics:
+  other: "Sujets&nbsp;"
+view-tags:
+  other: "Voir les billets balisés avec:"
```

Super minor, but should this have a non-breaking space like so?
```suggestion
  other: "Voir les billets balisés avec&nbsp;:"
```